README.md

```diff
@@ -10,7 +10,7 @@ To build the plugin run:
 $./gradlew build
```

Comment:
We still use the wrapper for building it?

Reply:
No. Fixing.